src/NServiceBus.AcceptanceTests/Serialization/When_receiving_interface_message_where_child_is_excluded.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace NServiceBus.AcceptanceTests.Serialization
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_receiving_interface_message_where_child_is_excluded : NServiceBusAcceptanceTest
+    {
+        static string ReceiverEndpoint => Conventions.EndpointNamingConvention(typeof(Receiver));
```

Comment:
nitpicking, but this doesn't need to use `=>`, `{get;} =` or just `=` would be more efficient.